packages/hash/frontend/src/pages/[account-slug]/types/entity-type/util.ts

```diff
@@ -39,31 +38,20 @@ export const withHandler = <
   T extends ReturnType<typeof bindTrigger> | ReturnType<typeof bindToggle>,
 >(
   trigger: T,
-  handler: () => void,
+  handler: undefined | (() => void),
```

Comment:
I see why you think so, but calling `withHandler` without passing an argument for handler is nonsensical, as the whole idea is to call a handler. The parameter isn't optional. The change is so that components which take an optional prop don't need to conditionally call `withHandler`.